proteus/mbd/CouplingFSI.pyx

```diff
@@ -1318,7 +1347,7 @@ cdef class ProtChBody:
         elements = ET.SubElement(topology,
                                  "DataItem",
                                  {"Format": dataItemFormat,
-                                  "DataType": "Int",
+                                  "DataType": "UInt",
```

Comment:
is this a python3 thing or a new version of chrono thing?

Reply:
Actually I am not sure that this has an effect but that was the syntax used for Polyline topology in one of the latest examples I found, so I used it here. It is an xmf / Paraview thing